src/twisted/application/internet.py

```diff
@@ -1029,13 +1058,31 @@ def __init__(self, endpoint, factory, retryPolicy=None, clock=None):
             this attribute will not be serialized, and the default value (the
             reactor) will be restored when deserialized.
         @type clock: L{IReactorTime}
+
+        @param prepareConnection: A single argument L{callable} that may return
```

Comment:
This is the one blocker:  the `prepareConnection` parameter needs a "since" declaration explaining when it's available since.  See https://github.com/twisted/incremental#updating for an explanation about how to phrase this so it will be picked up by the release machinery.